Modules/System/XmlWriter/src/XmlWriterImpl.Codeunit.al

```diff
@@ -19,6 +19,12 @@ codeunit 1484 "XmlWriter Impl."
         XmlTextWriter.WriteStartDocument();
     end;
 
+    procedure WriteProcessingInstruction(Encoding: Text)
+    begin
+        Initialize();
+        XmlTextWriter.WriteProcessingInstruction('xml', STRSUBSTNO('version="1.0" encoding="%1" standalone="no"', Encoding));
```

Comment:
Why not implement the actual [WriteProcessingInstruction](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlwriter.writeprocessinginstruction?view=net-5.0)?

Something like 
```
WriteProcessingInstruction(Name: Text, "Text": Text)
```

Reply:
Yes, that's better indeed. I have updated the code!